provider/resources.go

```diff
@@ -3342,6 +3342,30 @@ func Provider() tfbridge.ProviderInfo {
 					Source: "privileged_access_manager_entitlement.html.markdown",
 				},
 			},
+			"google_dataplex_aspect_type_iam_binding": {
+				Tok:  gcpResource(gcpDataPlex, "AspectTypeIamBinding"),
```

Comment:
Can you remove the `Tok` overrides and just use automapping?

Reply:
Seems like we have quite a few resources where we don't need this - I was following the convention here.

I'll raise a separate PR to remove the token overrides